upload-api/types.ts

```diff
@@ -290,6 +290,7 @@ export interface BillingProvider {
   hasCustomer: (customer: AccountDID) => Promise<Result<boolean, Failure>>
   setPlan: (customer: AccountDID, plan: DID) => Promise<Result<Unit, SetPlanFailure>>
   createAdminSession: (customer: AccountDID, returnURL: string) => Promise<Result<PlanCreateAdminSessionSuccess, PlanCreateAdminSessionFailure>>
+  createCheckoutSession: (customer: AccountDID, planID: PlanID, options: PlanCreateCheckoutSessionOptions) => Promise<Result<PlanCreateCheckoutSessionSuccess, PlanCreateCheckoutSessionFailure>>
```

Comment:
What is this change? Is this something updated in the deps that hasn't yet been implemented here? Seems unrelated to pdp/filecoin.